server.js

```diff
@@ -0,0 +1,9 @@
+const http = require('http');
```

Comment:
while this is correct, use express package to start a server.

return 'hello world' when hitting http://localhost:3030/